Adds detergent pods

comma opened the pull request:
Puts a bag of detergent pods in janitor's closet, put them in water to make space cleaner.

<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You find a README and example file in .\html\changelogs\ for further instructions.
-->

Spamcat commented:
inb4 someone posts this on r/ss13

mustafakalash commented on code/game/objects/items/weapons/storage/boxes.dm:
```diff
@@ -493,4 +493,10 @@
+
+/obj/item/weapon/storage/box/detergent
+	name = "detergent pods bag"
+	desc = "A bag full of juicy yummy detergent pods."
```
Unacceptable. It's "juicy, yummy." Use commas to separate adjectives.

Techhead0 replied:
Is 'detergent' also an adjective here or should 'detergent pod' be treated as a single noun?

Jovaniph replied:
detergent is surprisingly an adjective and I think is it acceptable to use a noun in place of one in a sentence, but it depends on the context.

mustafakalash commented:
That's a noun.

ghost commented:
...Can I eat it?

Spamcat commented:
why would you want to eat detergent pods tho?

Sbotkin commented:
Best meme RP of 2018. Merge this ASAP.

ghost commented:
>why would you want to eat detergent pods tho?

Why not?

Dukica99 commented:
Give this to your local, not-so-friendly GAS as a food present.

Also, add me in the screencap compilation, reddit

masterrbc commented:
Jesus... I mean the idea is good...

xales reviewed (commented):
I wish I didn't have to ask, but might be worth having it log to admins, as with other self-harm, if someone actually does eat these (because presumably it will be to death...)

mustafakalash commented:
Ingesting harmful chemicals does not admin log in any other instance, and if it did, it should be implemented on a much higher level -- not per-item.

xales commented:
Given the obvious potential for abuse this item has beyond something much more noticeable to get access to (since eating them is some meme thing now), I think it's appropriate for there to be some easy way for admins to see what's going on, in the manner of things like someone trying to put themselves in disposals being messaged to admins.

Sbotkin commented:
IIRC feeding someone already leaves a log. If you eat them, the chemicals are easily trackable through the VV.

xales commented:
Actually - should already be logged, since it's a pill subtype.

ghost commented:
@comma Will the OD on ammonia affect Vox? Because it's the only chemical that can heal their oxyloss and giving it such low OD would make it useless.

comma committed:
Adds detergent pods

Puts a bag of detergent pods in janitor's closet, put them in water to make space cleaner.
Makes ammonia more toxic too (OD at 5u now)
